docs/contributer_guide/install.rst

```diff
@@ -0,0 +1,56 @@
+Developer Installation
+======================
+
+For using ``ngshare``, see :doc:`../user_guide/install`.
+
+Install from GitHub
+----------------
+.. code:: bash
+
+    git clone https://github.com/lxylxy123456/ngshare.git
+    cd ngshare/
+    pip3 install .
+
+Run Installed ngshare
+---------------------
+.. code:: bash
+
+    python3 -m ngshare [arguments]
+
+Run ngshare without Installation
+--------------------------------
+The first line installs pip dependencies
+
+.. code:: bash
+
+    pip3 install tornado jupyterhub sqlalchemy
+    git clone https://github.com/lxylxy123456/ngshare.git
+    cd ngshare/ngshare/
+    python3 ngshare.py [arguments]
+
+Run vngshare
+------------
+vngshare is simply adding some arguments to ngshare.
+
+* ``--vngshare``: Mock authentication (using only username)
+* ``--debug``: enable debug
+* ``--database sqlite:////tmp/ngshare.db``: change default database path
+* ``--storage /tmp/ngshare``: change default storage path
+
+Run vngshare from Installed ngshare
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. code:: bash
+
+    python3 -m ngshare --vngshare --debug ... [arguments]
```

Comment:
What do the ellipses represent?

Reply:
`--database sqlite:////tmp/ngshare.db --storage /tmp/ngshare`

Reply:
should probably say that somewhere in the doc

Reply:
I think this is explained in the arguments right before it

Reply:
I meant we should say that the ellipses represent the remaining options from above.

Reply:
Can we just remove "..."?

Reply:
I removed the `...`